### DISCUSSION TITLE
How to get inference time for tracking algorithms?

### DISCUSSION THREAD

**Discussion by Bombex** [mikel-brostrom/yolo_tracking#1210] [Q&A] [answered]
Hello. Is there a table with inference times for many tracking algorithms in many variants? How to get fps?
For example: Detector + tracking algorithm + ReID (if it is possible) = FPS

**Comment by mikel-brostrom** [comment, marked as answer]
FPS is not given as it is highly dependent on underlying HW you run everything on.

**Reply by Bombex** [reply]
How can I measure FPS for my HW?

**Reply by mikel-brostrom** [reply]
It is also dependent on the number of objects to track. So the FPS will vary a lot based on how many objects there are in the field of view of the camera. You get the averages (ms) after running a video for example
